Assets/MixedRealityToolkit.Providers/WindowsMixedReality/WindowsMixedRealityEyeGazeDataProvider.cs

```diff
@@ -40,6 +40,8 @@ public class WindowsMixedRealityEyeGazeDataProvider : BaseInputDeviceManager, IM
 
         public bool SmoothEyeTracking { get; set; } = false;
 
+        public IMixedRealityEyeSaccadeProvider SaccadeProvider => this as IMixedRealityEyeSaccadeProvider;
```

Comment:
> as IMixedRealityEyeSaccadeProvider [](start = 70, length = 35)

I think this suffix isn't necessary right (unless you're explicitly writing it out for readability or something?)